backend/organization/views/organization_views.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.generics import ListAPIView
+from rest_framework.filters import SearchFilter
+from rest_framework.pagination import PageNumberPagination
+
+from organization.serializers.organization import OrganizationSerializer
+from organization.models import Organization
+
+
+class OrganizationAPIView(ListAPIView):
+    permission_classes = [IsAuthenticated]
```

Comment:
Here I'd do the same thing as with profiles: Users who aren't logged in should be able to partially view organization pages in my opinion.